modules/rostests/apitests/ntdll/NtQueryInformationProcess.c

```diff
@@ -209,30 +211,14 @@ Test_ProcessPriorityClassAlignment(void)
     RtlFillMemory(&ProcPriority->PriorityClass, sizeof(ProcPriority->PriorityClass), 0x55);
 
     /* Unaligned buffer -- wrong size */
-    Status = NtQueryInformationProcess(NtCurrentProcess(),
```

Comment:
why did you remove those tests?

Reply:
2-aligned checks fail, so 1-aligned checks seem to be useless.